changelog.d/12624.misc

```diff
@@ -0,0 +1,2 @@
+Remove use of constantly library and switch to enums for EventRedactBehaviour.
+Contributed by @andrewdoh.
```

Comment:
This needs to be a single line for towncrier to not break, I think.

```suggestion
Remove use of constantly library and switch to enums for EventRedactBehaviour. Contributed by @andrewdoh.
```

Reply:
If so, it'd be good to get that into the lints!